.github/workflows/publish-python-apidocs.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish Python API Docs
+
+# Run when the Python API changes or every month so that the artifact does not expire
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - onnxruntime_extensions/**
+      - docs/python/**
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+concurrency:
+  group: "apidocs-python"
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Generate Python API docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install tools
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip
+          sudo apt-get install graphviz
```

Comment:
why is this package needed?

Reply:
we can remove it, it's not needed now since we don't need any graphs for our api-docs.